### ISSUE TITLE
Missing new line in PyDoc

### ISSUE THREAD

**Issue by huckw** [swig/swig#1632]
Using Swig 4.0.1 and -doxygen the source file has the following documentation...

```
			/////////////////////////////////////////////////////////////////////
			/// @brief Compute slanted edge MTF
			/// Compute the slanted edge MTF's on an image containing multiple squares, each with four edges
			///
			/// @param TestImage image containing the squares
			/// @param OverSamplingFactor integer oversampling factor
			/// @param PixelPitch the pixel pitch in units of \todo {enter the pixel pitch units}
			/// @param side - either Left or Right (first letter must be upper-case)
			/// @param ResultsMTF the resulting MTF data structure
			/// @return true on success, false otherwise
			/////////////////////////////////////////////////////////////////////
			bool SlantedEdgeMTF(cv::Mat &TestImage, const float OverSamplingFactor, const float PixelPitch, const std::string Side, DataMTF &ResultsMTF);
```
And the generate .py docs look like this...
```
    def SlantedEdgeMTF(self, *args):
        r"""
        *Overload 1:*
         Compute slanted edge MTF Compute the slanted edge MTF's on an image containing multiple squares, each with four edges :type TestImage: :py:class:`Mat`
        :param TestImage: image containing the squares :type OverSamplingFactor: float
        :param OverSamplingFactor: integer oversampling factor :type PixelPitch: float
        :param PixelPitch: the pixel pitch in units of TODO: {enter the pixel pitch units} :param side: - either Left or Right (first letter must be upper-case) :type ResultsMTF: :py:class:`DataMTF`
        :param ResultsMTF: the resulting MTF data structure :rtype: boolean
        :return: true on success, false otherwise
        """
       
```
The `:type` and `:rtype` stuff is at the end of the previous line instead of on its own line.  Sometimes even a `:param` ends up on a previous line (like the "side" parameter)!  Am I doing something incorrectly?

**Comment by mcfarljm** [comment]
This appears to be an issue with Swig parsing the Doxygen comment style that you are using.  If you use this style:
```
/** 
 * comments
 */
```
then swig should process it correctly (the `:type` entries will come before the `:param` entries, each on their own line)

The reason that your example is missing the `:type` declaration for the `side` parameter is that there is a case mismatch between the Doxygen comment and the actual parameter declaration.

I will try to look into what is needed for swig to support this doxygen comment style, but in the meantime using the other style is a workaround.

**Comment by huckw** [comment]
Ah!  I didn't notice the mismatched case in side/Side....good catch.  I ended up writing a tool to add the newlines for me.  Not sure I can convince everyone on my team to switch to a different comment style...hopefully the other Doxygen comment styles will be supported in the future.

**Comment by mcfarljm** [comment]
This is fixed in PR #1659.

**Comment by wsfulton** [comment]
I confirmed since #1659 was merged the output is fixed and is now:
```python
def SlantedEdgeMTF(TestImage, OverSamplingFactor, PixelPitch, Side, ResultsMTF):
    r"""
    Compute slanted edge MTF
    Compute the slanted edge MTF's on an image containing multiple squares, each with four edges

    :type TestImage: :py:class:`Mat`
    :param TestImage: image containing the squares
    :type OverSamplingFactor: float
    :param OverSamplingFactor: integer oversampling factor
    :type PixelPitch: float
    :param PixelPitch: the pixel pitch in units of TODO: {enter the pixel pitch units}
    :param side: - either Left or Right (first letter must be upper-case)
    :type ResultsMTF: :py:class:`DataMTF`
    :param ResultsMTF: the resulting MTF data structure
    :rtype: boolean
    :return: true on success, false otherwise
    """
    return _example.SlantedEdgeMTF(TestImage, OverSamplingFactor, PixelPitch, Side, ResultsMTF)
```
